url-shortner-view/src/App.js

```diff
@@ -3,13 +3,16 @@ import './App.css';
 import Header from './Header/Header';
 import View from './View/View';
 import Footer from './Footer/Footer';
+import SecondPageView from './SecondPageView/SecondPageView';
+import { Route } from 'react-router-dom';
 
 function App() {
 	return (
 		<div className='App'>
-			<Header />
-			<View />
-			<Footer />
+			<Route path='/' component={Header} />
+			<Route path='/' exact component={View} />
+			<Route path='/shortener' component={SecondPageView} />
+			<Route path='/' component={Footer} />
```

Comment:
header, view and footer can be grouped into one component. No route should have more than one definition at the same time